### PR TITLE
Adopt centralized package versions stuff

### PR DESCRIPTION
**Pull request by j3parker**

### CODE REVIEW

File: Packages.props [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<ItemGroup>
+		<PackageReference Update="Newtonsoft.Json" Version="[6.0.6]" />
```
Comment (j3parker) [inline Packages.props:4]: Just starting with this one for now, I want a `-preview` build of stuff with 7.0.1 (follow-up PR)

Comment (j3parker) [reply Packages.props:4]: Oh, it's all or nothing eh

File: Packages.props [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<ItemGroup>
+    	<PackageReference Update="BenchmarkDotNet" Version="[0.12.1]" />
```
Comment (j3parker) [inline Packages.props:4]: Probably breaks dependabot :( I like how we do it in the LMS for now.

File: src/D2L.Security.OAuth2.WebApi/D2L.Security.OAuth2.WebApi.csproj [hunk 3]
```diff
@@ -1,4 +1,5 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+	<Sdk Name="Microsoft.Build.CentralPackageVersions" Version="2.0.1" />
```
Comment (JeffAshton) [inline src/D2L.Security.OAuth2.WebApi/D2L.Security.OAuth2.WebApi.csproj:2]: You can also centralize the version of this with `global.json`:

https://search.d2l.dev/xref/lms/global.json?r=66ab99aa

File: Packages.props (hunk 2 above)
Comment (JeffAshton) [reply Packages.props:4]: Visual studio will also do updates by setting the version in the projects, so people just need ot keep an eye out in reviews.  But the centralized is much nicer.

**Review by JeffAshton — APPROVED** [review]

File: src/D2L.Security.OAuth2.WebApi/D2L.Security.OAuth2.WebApi.csproj (hunk 3 above)
Comment (j3parker) [reply src/D2L.Security.OAuth2.WebApi/D2L.Security.OAuth2.WebApi.csproj:2]: Oh nice, will do.

**Pull request merged**